src/shadowbox/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "outline-server",
   "private": true,
-  "version": "1.5.0",
+  "version": "1.5.3",
```

Comment:
Why 1.5.3?
I think we should revert this.

Reply:
That's [where master is](https://github.com/Jigsaw-Code/outline-server/blob/master/src/shadowbox/package.json).  I'm not sure why this diff is displaying in this way.